packages/amqp-client-async/amqp-client-async.2.0.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: [ "Anders Fugmann" ]
+homepage: "https://github.com/andersfugmann/amqp-client"
+bug-reports: "https://github.com/andersfugmann/amqp-client/issues"
+dev-repo: "https://github.com/andersfugmann/amqp-client.git"
+license: "BSD3"
+version: "2.0.0"
```

Comment:
@andersfugmann I missed this.  For future reference and future releases, a `version` field isn't required as opam determines the version based on the directory structure.  That line can be dropped from future submissions to opam-repository.